docs/src/tut1.rst

```diff
@@ -50,17 +50,21 @@ as well as words that only appear once in the corpus:
     >>>
     >>> # remove common words and tokenize
     >>> stoplist = set('for a of the and to in'.split())
-    >>> texts = [[word for word in document.lower().split() if word not in stoplist]
-    >>>          for document in documents]
+    >>> texts = [
+    >>>     [word for word in document.lower().split() if word not in stoplist]
+    >>>     for document in documents
+    >>> ]
     >>>
     >>> # remove words that appear only once
     >>> frequency = defaultdict(int)
```

Comment:
I know this isn't part of the PR, but it may be better to simplify this section as:

```python
# untested code but "should" work
import collections
counter = collections.Counter()
for text in texts:
    counter.update(text)
```

Reply:
Fine with me.